examples/knative/powercli/kn-pcli-template/README.md

```diff
@@ -0,0 +1,208 @@
+# Knative Function Template 
+Delete this section before publishing your new function
+- Find all instances of #REPLACE-ME# and replace it with descriptive names for your function
+	- For example, if you are writing a function for a distributed virtual switch, you might name the function `kn-pcli-dvs`
+- Find all instances of FUNCTION_SECRET and function_secret, replacing it with descriptive names for your function. For example, if you are writing a function for a distributed virtual switch, you might use DVS_SECRET
+	- test/docker-test-env-variable
+	- handler.ps1
+	- README.md
+	- function_secret.json (rename the file)
+
+
+# kn-pcli-#REPLACE-ME#
+Example Knative PowerCLI function #REPLACE-ME#
+
+# Step 1 - Build
+
+> **Note:** This step is only required if you made code changes to `handler.ps1`
+> or `Dockerfile`.
+
+Create the container image locally to test your function logic.
+
+Mac/Linux
+```
+# change the IMAGE name accordingly, example below for Docker
+export IMAGE=<docker-username>/kn-pcli-#REPLACE-ME#:1.0
+docker build -t ${IMAGE} .
+```
+
+Windows
+```
+# change the IMAGE name accordingly, example below for Docker
+$IMAGE="<docker-username>/kn-pcli-#REPLACE-ME#:1.0"
+docker build -t ${IMAGE} .
+```
+# Step 2 - Test
+
+Verify the container image works by executing it locally.
+
+Change into the `test` directory
+```console
+cd test
+```
+
+Update the following variable names within the `docker-test-env-variable` file
```

Comment:
here we should also add a note to replace this variables if needed (since we can't assume it's always VC function)